cmake/caches/Windows-aarch64.cmake

```diff
@@ -26,21 +26,65 @@ set(LLVM_DEFAULT_TARGET_TRIPLE aarch64-unknown-windows-msvc CACHE STRING "")
 set(LLVM_APPEND_VC_REV NO CACHE BOOL "")
 set(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR YES CACHE BOOL "")
 set(LLVM_ENABLE_PYTHON YES CACHE BOOL "")
-set(LLVM_RUNTIME_TARGETS
+
+set(default_targets
+      x86_64-unknown-windows-msvc
       aarch64-unknown-windows-msvc
-    CACHE STRING "")
+      i686-unknown-windows-msvc)
+set(LLVM_RUNTIME_TARGETS ${default_targets} CACHE STRING "")
+
+# Build the android builtins if NDK path is provided.
+if(NOT "$ENV{NDKPATH}" STREQUAL "")
+  list(APPEND default_targets
+       aarch64-unknown-linux-android
+       x86_64-unknown-linux-android
+       i686-unknown-linux-android
+       armv7-unknown-linux-androideabi)
+endif()
+
+set(LLVM_BUILTIN_TARGETS ${default_targets} CACHE STRING "")
+
 foreach(target ${LLVM_RUNTIME_TARGETS})
   set(RUNTIMES_${target}_LLVM_ENABLE_RUNTIMES
         compiler-rt
       CACHE STRING "")
   set(RUNTIMES_${target}_CMAKE_MT mt CACHE STRING "")
   set(RUNTIMES_${target}_CMAKE_SYSTEM_NAME Windows CACHE STRING "")
   set(RUNTIMES_${target}_CMAKE_BUILD_TYPE Release CACHE STRING "")
+  set(RUNTIMES_${target}_COMPILER_RT_BUILD_BUILTINS YES CACHE BOOL "")
```

Comment:
I'm slightly worried about this as this is the "legacy" path - the builtins build I thought was being replaced with the runtimes build.

Reply:
My understanding was that this option lets the runtimes build include `llvm-project\compiler-rt\lib\builtins` which results in a `builtins-<triple>` build-tree next to the `runtimes-<triple>` one in the `runtimes` build directory. I don't see a deprecation warning in CMake, neither swiftlang not upstream  https://github.com/llvm/llvm-project/blob/main/compiler-rt/lib/CMakeLists.txt I might be wrong though.

Reply:
Ok, double-checked: I should should switch this off as we get duplicate build artifacts in:
```
runtimes\runtimes-x86_64-unknown-windows-msvc-bins\compiler-rt\lib\builtins
runtimes\builtins-x86_64-unknown-windows-msvc-bins
```